arm-servicefabric/2016-09-01/examples/ClusterPutOperation_exmaple.json

```diff
@@ -0,0 +1,323 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "resRg",
+    "clusterName": "myCluster",
+    "api-version": "2016-09-01",
+    "ClusterUpdateParameters": {
+      "properties": {
+        "availableClusterVersions": [
+          {
+            "codeVersion": "5.1.158.9590",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.2.0.100",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.2.207.9590",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.3.121.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.3.311.9590",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.30.100",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.35.100",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.38.100",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.118.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.132.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.145.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.164.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.5.67.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.5.149.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          }
+        ],
+        "clusterId": "4d0af3c4-04c4-4ea4-a7a0-5917aca0fe03",
+        "clusterState": "Ready",
```

Comment:
clusterState is readonly in the swagger and in put you are passing it? Please make sure you provide right examples and right field attribute.